Resources/views/CRUD/base_list.html.twig

```diff
@@ -295,7 +295,13 @@ file that was distributed with this source code.
                         </div>
 
                         {% for paramKey, paramValue in admin.persistentParameters %}
-                            <input type="hidden" name="{{ paramKey }}" value="{{ paramValue }}">
+                            {% if paramValue is iterable %}
+                                {% for key, value in paramValue %}
+                                    <input type="hidden" name="{{ paramKey }}[{{ key }}]" value="{{ value }}">
+                                {% endfor %}
```

Comment:
Did you get a response about that? I can't find an example of test on a twig file.